pkgs/applications/misc/taskell/default.nix

```diff
@@ -0,0 +1,59 @@
+{ haskell, lib, haskellPackages, fetchFromGitHub }:
+
+let
+  version = "1.3.2";
+  sha256  = "0cyysvkl8m1ldlprmw9mpvch3r244nl25yv74dwcykga3g5mw4aa";
+
+in (haskellPackages.mkDerivation {
+  pname = "taskell";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "smallhadroncollider";
+    repo = "taskell";
+    rev = version;
+    inherit sha256;
+  };
+
+  postPatch = ''${haskellPackages.hpack}/bin/hpack'';
+
+  # basically justStaticExecutables; TODO: use justStaticExecutables
```

Comment:
Dunno what we should do about this. I tried for about 5 minutes to get it to run with `haskellPackages.callPackage`, but just copying the arguments was easier. Would be nicer if we could use `justStaticExecutables`.